sql/updates/world/4.3.4/2021_07_23_00_world.sql

```diff
@@ -0,0 +1,360 @@
+REPLACE INTO `quest_offer_reward` VALUES (26965, 274, 1, 273, 0, 0, 0, 0, 0, 'Well that was unexpected. I am not one to doubt our Warchief, but...$B$BNevermind! We have work to do. I\'m sure the Dark Lady and the Warchief will get this all worked out.', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (26992, 5, 1, 273, 0, 0, 0, 0, 0, 'Agony abounds, friend! The land itself threatens to devour us, and yet we persist. I wonder why that is? With this doomweed and the guts and gore that T\'Veen needs, we might have something suitable to send to the Forsaken front.', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (26995, 4, 1, 0, 0, 0, 0, 0, 0, 'Excellent! With these guts and the ferocious doomweed we\'ll be able to test out our new strain of blight. Now to select a suitable target...', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (26989, 1, 5, 21, 21, 0, 0, 0, 0, '$Nnub has returned! And in one piece!$B$BDefender of Silverpine! Redeemer of the Horde! $Nnub, your bravery must be rewarded!', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (26998, 5, 1, 273, 0, 0, 0, 0, 0, 'Potent enough to exterminate a thousand murlocs! I think I\'ll put that on the label. You\'ve done well, $N, and great work around here gets rewarded.', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (27039, 0, 0, 0, 0, 0, 0, 0, 0, 'Belmont gets a little nervous and sends some newbie to look for me? Listen, I\'ve been staking out the old Ivar cottage from inside this outhouse for three days. Your presence threatens the whole damn operation!$B$BThere\'s no time to explain. We have to act now or this mission will be compromised.', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (27045, 5, 1, 0, 0, 0, 0, 0, 0, 'Alas, poor Yorick! I knew her, $N. An assassin of infinite courage...$B$BNow, tell me what\'s going on? What did you and Yorick discover?', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (27056, 1, 1, 274, 5, 0, 0, 0, 0, 'Crowley... A gnat. An insect, waiting to be crushed beneath my boot heel.$B$BSo he thinks to rouse the feral worgen of Silverpine? We shall show him and his allies the might of the Forsaken!$B$BBut first we must clean up the mess made by the Warchief\'s fleet. Without their aid we may lose this war.', 15595);
+REPLACE INTO `quest_offer_reward` VALUES (27065, 1, 2, 0, 0, 0, 0, 0, 0, 'This fiasco may go down as one of the biggest embarrassments in the history of naval warfare. How did this happen? Drunk happened. That\'s how.', 15595);
+
+INSERT INTO `spell_area` VALUES (83989, 5369, 26964, 27097, 0, 0, 2, 3, 74, 11);
+INSERT INTO `spell_area` VALUES (83232, 5369, 26964, 27098, 0, 0, 2, 3, 66, 1);
+
+INSERT INTO `spell_script_names` VALUES (83173, 'spell_silverpine_raise_forsaken_83173_spell');
+INSERT INTO `spell_script_names` VALUES (83149, 'spell_silverpine_forsaken_trooper_master_script_83149');
+
+INSERT INTO `spell_target_position` VALUES (83756, 0, 0, 1314.91, 1211.39, 58.4967, 5.8, 0);
```

Comment:
where did that value come from?

Reply:
The spell itself uses a DB value, and it was missing from the database. So I just used a parsed sniff Joker handed on to me.

Reply:
Removed since it was already on the DB.